ucan/src/ucan.rs

```diff
@@ -135,7 +137,11 @@ impl Ucan {
 
     /// Returns true if this UCAN expires no earlier than the other
     pub fn lifetime_ends_after(&self, other: &Ucan) -> bool {
-        self.payload.exp >= other.payload.exp
+        match (self.payload.exp, other.payload.exp) {
```

Comment:
This probably deserves a quick unit test, WDYT?

Reply:
[This test](https://github.com/ucan-wg/rs-ucan/pull/95/files#diff-6d328ec1debdfbe28c2a86236235e43efe720cd06a25f6c079f74e423a2d9c23R164) probably should just be rolled into other ucan tests (rather than a spec-pinned section). Can explicitly add the `(Some, Some)` case to test as well